chat-widget/src/common/Constants.ts

```diff
@@ -119,6 +119,7 @@ export class Constants {
 
     public static readonly ChatWidgetStateChangedPrefix = "ChatWidgetStateChanged";
     public static readonly PostChatLoadingDurationInMs = 2000;
+    public static readonly BrowserUnloadConfirmationMessage = "Do you want to leave chat?";
```

Comment:
Is this customizable? We need localization on this text

Reply:
Browsers have their default prompts and don't accepts customized string. This is just a dummy string for browsers to prompt the confirmation.

Reply:
![image](https://user-images.githubusercontent.com/45749980/187304949-65532743-5324-4069-9caa-d4f752552543.png)